lib/tasks/readyset.rake

```diff
@@ -1,13 +1,87 @@
 require 'colorize'
 require 'erb'
 require 'progressbar'
+require 'terminal-table'
 
 namespace :readyset do
+  desc 'Creates a cache from the given query ID'
+  task :create_cache, [:id] => :environment do |_, args|
+    if args.key?(:id)
+      Readyset.create_cache!(id: args[:id])
+    else
+      puts 'A query ID must be passed to this task'
```

Comment:
**question:_** Should we use `Rails.logger` in addition to the terminal messages?

Reply:
Yup, I think you're right! I'll update that